resolwe_bio/tools/tximport_summarize.R

```diff
@@ -25,7 +26,18 @@ if (args$ignoreTxVersion) {
 write.table(tx2gene, file=args$gene_mapping, sep="\t", quote=FALSE, row.names=FALSE)
 
 # Run tximport
-txi_tpm = tximport(args$infile, type="salmon", tx2gene=tx2gene, ignoreTxVersion=args$ignoreTxVersion)
+if (args$variance) {
+    txi_tpm = tximport(args$infile, type="salmon", tx2gene=tx2gene, varReduce = TRUE,
+                    ignoreTxVersion=args$ignoreTxVersion)
+
+    tpm_variance = data.frame(Variance = txi_tpm$variance)
+    tpm_variance = cbind(FEATURE_ID = rownames(tpm_variance), tpm_variance)
+    rownames(tpm_variance) = NULL
```

Comment:
Since you're using `row.names = FALSE` to write to a file, removing row names may not be necessary.